web/src/components/layout/Icon.jsx

```diff
@@ -99,13 +109,17 @@ const icons = {
   settings_applications: SettingsApplications,
   settings_ethernet: SettingsEthernet,
   signal_cellular_alt: SignalCellularAlt,
+  storage: Storage,
   task_alt: TaskAlt,
   terminal: Terminal,
   translate: Translate,
   tune: Tune,
   warning: Warning,
   wifi: Wifi,
-  wifi_find: WifiFind
+  wifi_find: WifiFind,
+  // brand icons
+  linux_logo: SiLinux,
+  windows_logo: SiWindows
```

Comment:
np: It should not block us but, do we have considered MacOS?

Reply:
Ah, thanks for remind me this. I wanted to ask the same to @joseivanlopez while I was writing the tests... but I forget to do so.

Reply:
Well, I think we are not able to detect MacOS systems yet.